pkgs/top-level/all-packages.nix

```diff
@@ -9571,6 +9571,13 @@ in
     buildPackages = buildPackages // { stdenv = gcc8Stdenv; };
   });
 
```

Comment:
Can you add `buildGo116{Module,Package}` please?

Reply:
Done